gilbreth_gazebo/scripts/run_gilbreth_test_sim.bash

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+export GAZEBO_MODEL_PATH="$(rospack find gilbreth_gazebo)/models"
+echo $GAZEBO_MODEL_PATH
+
+roslaunch gilbreth_gazebo gilbreth.launch
```

Comment:
The `_test` portion should be removed from the script name if this isn't a test but the actual application.